### DISCUSSION TITLE
skin editor bug

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#17623] [Q&A] [answered]
step to replicate:
1. press ctrl+shift+s to open the skin editor
2. do whatever, then close the editor by hitting exit on the toolbar
3. inputs fail to affect the game until the next time you hit ctrl+shift+s, which unlocks the input and doesn't reopen the editor

version: 2022.327.0

**Comment by integerrr** [comment]
Was going to make an issue thread and post the following but saw this instead. 

https://user-images.githubusercontent.com/7279624/161407768-74617a42-2517-4bdb-b12e-ab071c760eb5.mp4

[database.log](https://github.com/ppy/osu/files/8403647/database.log)
[network.log](https://github.com/ppy/osu/files/8403648/network.log)
[performance.log](https://github.com/ppy/osu/files/8403649/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8403650/runtime.log)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/17584
#17489 
#17542
https://github.com/ppy/osu/pull/17506
